files/image_config/hostcfgd/hostcfgd

```diff
@@ -49,6 +50,74 @@ def obfuscate(data):
     else:
         return data
 
+class Iptables(object):
+    def __init__(self):
+        '''
+        Default MSS to 1460 - (MTU 1500 - 40 (TCP/IP Overhead))
+        '''
+        self.tcpmss = 1460
```

Comment:
noticed that you also did for ipv6, v6 header is 40 bytes so total 60 bytes and the payload is only 1440 bytes?

Reply:
That's a good find! RFC states "When using TCP over IPv6, the MSS must be computed as the maximum packet size minus 60 octets.”. I'll make the change and resubmit.